tests/wake-format/basic/stdout

```diff
@@ -211,6 +227,7 @@ def t10 =
 def name Unit = # wake-format off
     def other = here
     def other = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+
     another
```

Comment:
Should disabling the formatting disable the newline, or am I misunderstanding where the `# wake-format off` is bound to?

Reply:
I'm unclear, we don't expect anyone to use it like this so I don't think it matters but I think this is binding to the left so its saying "don't format the '=' token" lol. There's no place to bind the format off comment to the block itself so you'd have to turn off formatting for the whole def I think.

Reply:
This pushes in to the just the first def.
Its the same as 

```
def name Unit =
     # wake-format off
    def other = here
    def other = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"

    another
```

so only the first def is "off", everything is formatted normally. FWIW if I see this is code review I'd ask it to be changed. This is more a sanity check than anything else